task.go

```diff
@@ -260,6 +260,13 @@ func (t *Task) Modify(query Query) {
 	if query.Priority != "" {
 		t.Priority = query.Priority
 	}
+
+	if query.Note != "" {
```

Comment:
I think the if statements can be combined, as if query.Note is `""` when appending it to `t.Notes` will have no effect so it doesn't have to be conditional; the only thing that needs to be is the addition of `"\n"`.

Reply:
Other than that looks good to go. Well tested too. Thanks!